person.rb

```diff
@@ -1,9 +1,13 @@
-class Person
+require_relative 'nameable'
+require_relative 'rental'
+
+class Person < Nameable
   attr_accessor :name, :age
```

Comment:
- [ ] Getter and setter should be present for the rentals too this is for the data to be read and writeable. Remember to add it as an array to it in the initialize because it would hold the data